src/python/tempusloader/provider/config.py

```diff
@@ -28,8 +28,37 @@
 # Path where binaries are located (used below)
 BINPATH=""
 
-import sys
+# PostgreSQL's console client binary location
+PSQL=BINPATH + ""
 
-PSQL="psql" + (".exe" if sys.platform.startswith('win') else "")
-SHP2PGSQL="shp2pgsql" + (".exe" if sys.platform.startswith('win') else "")
+# PostGIS shp2pgsql tool
+SHP2PGSQL=BINPATH + ""
 
```

Comment:
This is apparently not the last version, which have been simplified and fixed for windows